### ISSUE TITLE
Springs and high triangle collision surfaces

### ISSUE THREAD

**Issue by Woovie** [godot-jolt/godot-jolt#764]
Having an issue that I'm not entirely sure how to work around nor the actual cause. When using RigidBody3D with JoltGeneric6DOFJoint3D to build a vehicle, I'm experiencing bumpy response on my springs in combination with surfaces generated from Terrain3D.

Here's a video of the issue: https://youtu.be/UJ2mkj5iNHk

Here is a full project I built out to test this: https://drive.google.com/file/d/1NrZustmjPDWhwhlTJW0_hugiSTIhxISU/view?usp=sharing

I apologize if I missed something obvious or a setting I can use to fix this. I'd be glad to create a video when I'm done if it is something simple to help educate others.

Thank you

**Comment by Woovie** [comment]
I did some more testing and tried setting Physics ticks per second to 120 and reducing Contact Speculative Distance and Contact Allowed Penetration to 0. This did help, but I continue to replicate regardless.

**Comment by Woovie** [comment]
And of course, 10 days ago this was posted, so I think it's a known and future solved issue. Just waiting on upstream :)

https://twitter.com/jrouwe/status/1748821475623088550

**Comment by mihe** [comment]
Yes, I haven't had a chance to test your project yet, but this is likely related to #587, and could potentially be resolved by Jolt's new internal edge removal.

**Comment by Woovie** [comment]
Definitely related to that issue. Sadly the parameters in that thread didn't help much, but I look forward to that change coming in. Thanks for your efforts. I am fine closing this given a fix is on the way.

**Comment by mihe** [comment]
This new "Use Enhanced Internal Edge Removal" project setting (which is enabled by default) can now be found in the latest pre-release, 0.13.0-beta1, which you'll find [here](https://github.com/godot-jolt/godot-jolt/releases).

Note that this version targets the just recently released [Godot 4.3-beta1](https://godotengine.org/article/dev-snapshot-godot-4-3-beta-1/), so won't work with Godot 4.2. Also note that this version is not available through the Godot Asset Library, so you'll need to download it through GitHub.

A proper release will be out some time around Godot 4.3-stable.

**Comment by mihe** [comment]
I figured I'd give a minor update here, as [0.13.0-stable](https://github.com/godot-jolt/godot-jolt/releases/tag/v0.13.0-stable) has just been released, so if you've been relying on 0.13.0-beta1 for this functionality until now I would highly recommend updating, as this most recent release includes some important bug fixes.

Note that the release on the Godot Asset Library is still pending approval as of typing this, so you might need to download it from the link above.

**Comment by Woovie** [comment]
Thank you, I'll try it out soon, been really busy but your efforts are greatly appreciated!
